tests/unit/test_unpack.py

```diff
@@ -183,6 +183,41 @@ def test_topic():
     validate(command, message, expected)
 
 
+def test_channelmode_no_params():
+    """ MODE command """
+    command = "CHANNELMODE"
+    expected_kwargs = {"channel": "#ch", "host": "",
+                       "modes": "+m", "params": ""}
+    message = "MODE #ch +m"
+    assert (command, expected_kwargs) == unpack_command(message)
+    expected_kwargs['user'] = 'usr'
+    expected_kwargs['nick'] = 'nck'
+    assert set(expected_kwargs) == set(parameters(command))
+
+
+def test_channelmode():
+    """ MODE command """
+    command = "CHANNELMODE"
+    expected_kwargs = {"channel": "#ch", "host": "",
+                       "modes": "+o", "params": ['trget']}
+    message = "MODE #ch +o trget"
```

Comment:
Please make this test expect multiple params, eg:

```MODE #ch +o trget trget2```

should generate

```kwargs["params"] == ["trget", "trget2"]```

Reply:
No problem.